IZT6ZK/Commands/CreateQuestionCommand.cs

```diff
@@ -0,0 +1,233 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using IZT6ZK.Assists;
+using IZT6ZK.Db;
+using IZT6ZK.StateMachine;
+
+namespace IZT6ZK.Commands;
+
+internal class CreateQuestionCommand : ICommands
+{
+    public void Execute()
+    {
+        DbManager dbManager = new DbManager();
+        string? input = null;
+        List<string>? inputQuestionAndAnswers = new List<string>();
+
+        string? inputWantTopic = null;
+        string? inputTopicId = null;
+        int wantedTopicId = 0;
+
+        bool loopGoing = true;
+        var stateQuestionReading = CreateQuestionStateMachine.QuestionReading;
+
+        ConsoleHelper.WriteQuit();
+
+        while (loopGoing)
+        {
+            switch (stateQuestionReading)
+            {
+                case CreateQuestionStateMachine.QuestionReading:
+                    input = ConsoleHelper.ReadAndWrite("your question");
```

Comment:
Huhh, It a Case minden ága lehetne egy külön metódus is akár, mert ez így baromira átláthatatlan.